README.md

```diff
@@ -76,6 +77,28 @@ var app = new EmberApp(defaults, {
 
 The function receives the `filePath` for each asset and must return a string.
 
+### Using With `broccoli-asset-rev`
```

Comment:
Okay, seems fine then. I'm not super against and as someone who is super close to the project, I am likely less attuned to what the readers need.